internal/resources/package/scope/scope.go

```diff
@@ -48,12 +48,14 @@ func ConstructScope(d *schema.ResourceData, name, metadataName, namespace string
 		if clusterValue, ok := clusterData.([]interface{}); ok && len(clusterValue) != 0 {
 			scopedFullnameData = &ScopedFullname{
 				Scope:           commonscope.ClusterScope,
-				FullnameCluster: ConstructClusterPackageFullname(clusterValue, name, metadataName, namespace),
+				FullnameCluster: ConstructClusterPackageFullname(clusterValue, name, metadataName),
 			}
+
+			scopesFound = append(scopesFound, commonscope.ClusterKey)
 		}
 	}
 
-	return scopedFullnameData
+	return scopedFullnameData, scopesFound
```

Comment:
Is there any other way to implement this without returning `scopesFound`?

Reply:
for data source this is what we have decided